lib/foreman_maintain/cli/content_command.rb

```diff
@@ -35,8 +35,11 @@ def execute
       end
 
       subcommand 'remove-pulp2', 'Remove pulp2 and mongodb packages and data' do
+        interactive_option(['assumeyes'])
```

Comment:
There is addition of `plaintext` flag recently, though we dont have lots of spinner I feel it will help to get automation friendly output if required?